opm/autodiff/VFPHelpersLegacy.hpp

```diff
@@ -0,0 +1,240 @@
+/*
+  Copyright 2015 SINTEF ICT, Applied Mathematics.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef OPM_AUTODIFF_VFPHELPERSLEGACY_HPP_
+#define OPM_AUTODIFF_VFPHELPERSLEGACY_HPP_
+
+
+#include <opm/parser/eclipse/EclipseState/Schedule/VFPProdTable.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/VFPInjTable.hpp>
+#include <opm/autodiff/AutoDiffHelpers.hpp>
+#include <opm/material/densead/Math.hpp>
+#include <opm/material/densead/Evaluation.hpp>
```

Comment:
I do not think the legacy code needs the things from `densead`

Reply:
indeed, copy-paste issue. will remedy after jenkins has run.

Reply:
remedied now. cancelled the old build and retriggered, since there was a wait anyways..